packages/slate/src/interfaces/point.ts

```diff
@@ -85,10 +84,8 @@ export const Point: PointInterface = {
    */
 
   isPoint(value: any): value is Point {
-    return (
-      isPlainObject(value) &&
-      typeof value.offset === 'number' &&
-      Path.isPath(value.path)
+    return Boolean(
+      value && typeof value.offset === 'number' && Path.isPath(value.path)
```

Comment:
I'm checking for existence here because `isRange` will pass in potentially-undefined points. Should I check for existence everywhere?